contracts/utils/contracts/test/TestLogDecoding.sol

```diff
@@ -1,43 +0,0 @@
-/*
```

Comment:
I don't see any `LogDecoding`. Is this test still in use?

Reply:
I think this was testing decoding from `ethereum-types` library rather than any protocol logic https://github.com/0xProject/protocol/pull/643/commits/e0873b716b88d59a5ad245626547148cc8d439fb#diff-f84b381c715449735700973aa5cf405d5ebdecf1c6fcc7c0fa261b847e5cf476L5
if that's what you meant?